cmake/modules/libsinsp.cmake

```diff
@@ -11,31 +11,41 @@ option(WITH_CHISEL "Include chisel implementation" OFF)
 
 include(ExternalProject)
 include(libscap)
-include(tbb)
+if(NOT WIN32)
+	include(tbb)
+endif()
 include(jsoncpp)
 if(NOT MINIMAL_BUILD)
 	include(cares)
 endif()
-include(b64)
-include(jq)
 
-set(LIBSINSP_INCLUDE_DIRS ${LIBSINSP_DIR}/userspace/libsinsp ${LIBSCAP_INCLUDE_DIRS} ${DRIVER_CONFIG_DIR})
+if(NOT WIN32)
+	include(b64)
+	include(jq)
+endif()
```

Comment:
Any chance this could be in a single `if` with tbb? (i.e. move the includes around a bit)

Reply:
Sure, done.